model/generate.go

```diff
@@ -151,7 +147,9 @@ func (g *GeneratedProject) NewVersion() (*Project, *Version, *task.Task, *projec
 			gimlet.ErrorResponse{StatusCode: http.StatusBadRequest, Message: errors.Wrap(err, "error creating config from generated config").Error()}
 	}
 	v.Config = newConfig
-	if err := LoadProjectInto([]byte(v.Config), t.Project, p); err != nil {
+	v.ParserProject = nil // ensure we read from new config string
```

Comment:
I don't follow what the purpose of this is. I think in generate.tasks you should be able to deal entirely with the `parserProject`.